src/pages/juyoung/Login/Login.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import "./Login.scss";
+import { Link, withRouter } from 'react-router-dom' ;
+
+
+class Login extends React.Component{
+    
+    goToMain = () => {
+       this.props.history.push('/Main-Juyoung');
+    }
+
+    constructor(props){
+      super(props);
+        this.state = {
+          id : '',
+          pw : '',
+          active : false,
+
+        };
+    }
+
+    handleIdInput = (e) => {
+      console.log(e.target.value);
+      this.setState({
+        id : e.target.value,
+      });
+    };
+
+
+    handlePasswordInput = (e) => {
+      console.log(e.target.value);
+      this.setState({
+        pw : e.target.value,        
+      })
+    }
```

Comment:
```suggestion
    handleInputs = e => {
      this.setState({
        [e.target.name] : e.target.value,
      });
    };
```
위와 같이 id와 pw를 분리하지 않고 하나의 함수로 사용할 수 있습니다.
단, target이 되는 input에 각각 name 값을 넣어줘야 합니다!

Reply:
넵 수정하였습니다!